### ISSUE TITLE
theneweuropean.co.uk: detection

### ISSUE THREAD

**Issue by rafikiphoto** [uBlockOrigin/uAssets#20122]
### Prerequisites

- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] This is NOT a [YouTube](https://www.github.com/uBlockOrigin/uAssets/issues/19976), [Facebook](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [Twitch](https://www.github.com/uBlockOrigin/uAssets/issues/5184) report.
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/124740436/235392297-c4fc6290-73a6-4a70-96a9-3f6a7683f21d.png), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://www.theneweuropean.co.uk/`

### Category

detection

### Description

A banner telling me to disable ad-blocker extensions or add them to the trusted list.

![image](https://github.com/uBlockOrigin/uAssets/assets/28431654/5b5ca450-781b-4b6b-9569-729564e7b1ca)

### Other extensions used

none

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.52.2
Firefox: 118
filterset (summary):
 network: 105931
 cosmetic: 40030
 scriptlet: 18609
 html: 1069
listset (total-discarded, last-updated):
 default:
  user-filters: 53-1, never
  ublock-filters: 35832-135, now
  ublock-badware: 7275-0, now
  ublock-privacy: 591-0, now
  ublock-unbreak: 2170-0, now
  ublock-quick-fixes: 220-1, now
  easylist: 71854-692, now
  easyprivacy: 33186-720, now
  urlhaus-1: 12425-0, now
  plowe-0: 3729-1, now
filterset (user): [array of 54 redacted]
trustedset:
 added: [array of 1 redacted]
hostRuleset:
 added: [array of 1 redacted]
userSettings:
 advancedUserEnabled: true
hiddenSettings: [none]
supportStats:
 allReadyAfter: 545 ms (selfie)
 maxAssetCacheWait: 130 ms
popupPanel:
 blocked: 1
 network:
  googletagmanager.com: 1
```
</details>

**Comment by stephenhawk8054** [comment]
Which country are you when accessing the site?

**Comment by 3kh0** [comment]
tl;dr Not able to reproduce

I set my VPN to Madrid and browsed a few articles, other than getting giant banners to sign up for an account, I did not get a notice about using an Adblocker. I did verify that I did not have any other programs that would block ads. 

Config info: https://pastebin.com/xF7dGhq5

**Comment by rafikiphoto** [comment]
When you mentioned location I switched off my VPN (set to Spain Madrid) (my location is Seville) and the banner disappeared and let me sign in. I then reconnected my VPN, same server, and no banner after signing in. Odd.

Thanks for looking.

B.

------ Original Message ------
From "Echo" ***@***.***>
To "uBlockOrigin/uAssets" ***@***.***>
Cc "rafikiphoto" ***@***.***>; "Author" ***@***.***>
Date 17/10/2023 16:10:27
Subject Re: [uBlockOrigin/uAssets] theneweuropean.co.uk: detection (Issue #20122)

**Comment by stephenhawk8054** [comment]
You can check if your VPN has options of blocking ads/trackers or not.

Since none of us can reproduce the problem, I'll close the issue. Just continue here if you have further update.
